core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -161,10 +161,14 @@ class AdminClient(val time: Time,
     if (group.protocolType != ConsumerProtocol.PROTOCOL_TYPE)
       throw new IllegalArgumentException(s"Group ${groupId} with protocol type '${group.protocolType}' is not a valid consumer group")
 
-    group.members.map {
-      case member =>
-        val assignment = ConsumerProtocol.deserializeAssignment(ByteBuffer.wrap(member.assignment))
-        new ConsumerSummary(member.memberId, member.clientId, member.clientHost, assignment.partitions().asScala.toList)
+    if (group.state == "Stable") {
+      group.members.map {
+        case member =>
```

Comment:
The case is redundant here, please remove it. Also the parameter name should be in the previous line.